### ISSUE TITLE
Dockerfile 

### ISSUE THREAD

**Issue by rmokros** [signals-dev/Orion#155]
* Orion version: 0.1.5 
* Python version: 3.6
* Operating System: docker file ..

### Description

I run Tulog notebook

"CELL"
from orion import Orion

orion = Orion(
    pipeline='tadgan.json'
)

"RESULT"

Exception caught building MLBlock orion.primitives.tadgan.TadGAN
Traceback (most recent call last):
  File "/usr/local/lib/python3.6/site-packages/mlblocks/mlpipeline.py", line 113, in _build_blocks
    block = MLBlock(primitive, **block_params)
  File "/usr/local/lib/python3.6/site-packages/mlblocks/mlblock.py", line 159, in __init__
    primitive = load_primitive(primitive)
  File "/usr/local/lib/python3.6/site-packages/mlblocks/discovery.py", line 259, in load_primitive
    raise ValueError("Unknown primitive: {}".format(name))
ValueError: Unknown primitive: orion.primitives.tadgan.TadGAN

looks like primitive: orion.primitives.tadgan.TadGAN is not there .....

**Comment by sarahmish** [comment]
Hi @rmokros

Thank you for raising this issue, that was definitely not intended!

The docker image does not install the package into the python environment, but rather runs the local version of Orion (the cloned one) directly, which is causing an issue to the identification of primitive paths. To solve this, we should add an entry point to the local version installed on the Docker. 

As a temporary solution please add the following code block to the notebook you are running
```python3
import os
import mlblocks

base = os.path.dirname(os.path.dirname(os.getcwd())) # get the base directory of the docker
path = os.path.join(base, 'orion', 'primitives', 'jsons') #orion primitive path

mlblocks.add_primitives_path(path)
```

Let me know if you were able to run the notebook successfully with the above solution

**Comment by sarahmish** [comment]
Closing since this has been addressed in PR #163.

**Comment by ellofa** [comment]
Hello, I am trying to execute this part of code on the orion-ml guide page : 

from mlstars import load_primitive
primitive = load_primitive('orion.primitives.tadgan.TadGAN',
                           arguments={"X": X, "y": X, "epochs": 2, "batch_size": 1,
                                      "iterations_critic": 1})

but I keep having this error : Unknown primitive: orion.primitives.tadgan.TadGAN; i tried the suggested method in the comment above with mlblocks but it still does not work.
I appreciate your help!
